src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-event-buffer-time.mdx

```diff
@@ -30,21 +30,21 @@ freshnessValidatedDate: 2023-07-20
         <TabsBarItem id="cordova">
             Cordova
         </TabsBarItem>
-        <TabsBarItem id="maui">
-            .NET MAUI
-        </TabsBarItem>
         <TabsBarItem id="flutter">
             Flutter
         </TabsBarItem>
+        <TabsBarItem id="maui">
```

Comment:
@austin-schaefer I just realized that if I click on .NET MAUI, it takes me to Flutter. Also, we need some code formatting in the Flutter example.